roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -0,0 +1,100 @@
+#include "RConfigure.h"
+#include "TEnv.h"
+#include "TSystem.h"
+
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <exception>
+
+/**
+\file Initialisation.cxx
+\ingroup Roofitcore
+
+Run static initialisers on first load of RooFitCore.
+**/
+
+namespace {
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/// Print RooFit banner.
+void doBanner() {
+#ifndef __ROOFIT_NOBANNER
+  if (gEnv->GetValue("RooFit.Banner", 1) == 0)
+    return;
+
+  /// RooFit version tag.
+  constexpr char VTAG[] = "3.60";
+
+  std::cout << '\n'
+      << "\033[1mRooFit v" << VTAG << " -- Developed by Wouter Verkerke and David Kirkby\033[0m " << '\n'
+      << "                Copyright (C) 2000-2013 NIKHEF, University of California & Stanford University" << '\n'
+      << "                All rights reserved, please read http://roofit.sourceforge.net/license.txt" << '\n'
+      << std::endl;
+#endif
+}
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/// Inspect cpu capabilities, and load a architecture-specific libraries for RooFitCore/RooFit computations.
+void loadComputeLibrary() {
+  std::vector<std::string> libNames;
+//  libNames.push_back("libRooFitCoreCompute");
+//  libNames.push_back("libRooFitCompute");
+#ifdef R__HAS_MATHMORE
+//  libNames.push_back("libRooFitMoreCompute");
+#endif
+//  const auto loadHF = gSystem->Load("libHistFactory");
+//  if (loadHF == 0 || loadHF == 1)
+//    libNames.push_back("libHistFactoryCompute");
```

Comment:
I see `libNames` is empty for now.  I think it's Manos' job to do something with this...

Reply:
That's correct.